superset-frontend/src/views/CRUD/welcome/SavedQueries.tsx

```diff
@@ -57,19 +64,45 @@ interface SavedQueriesProps {
   mine: Array<Query>;
 }
 
+export const CardStyles = styled.div`
+  cursor: pointer;
+  a {
+    text-decoration: none;
+  }
+  .ant-card-cover {
+    border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
+  }
+`;
+
 const QueryData = styled.div`
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  border-bottom: 1px solid ${({ theme }) => theme.colors.grayscale.light2};
-  .title {
-    font-weight: ${({ theme }) => theme.typography.weights.normal};
-    color: ${({ theme }) => theme.colors.grayscale.light1};
+  svg {
+    margin-left: ${({ theme }) => theme.gridUnit * 10}px;
   }
-  .holder {
-    margin: ${({ theme }) => theme.gridUnit * 2}px;
+  .query-title {
+    padding: ${({ theme }) => theme.gridUnit * 2 + 2}px;
+    font-size: ${({ theme }) => theme.typography.sizes.l}px;
   }
 `;
+
+const ShowEmptyState = styled.div`
+  background-image: url('/static/assets/images/emptyquery.svg');
+  width: 100%;
+  height: 171px;
+  background-repeat: no-repeat;
+  background-size: 539px;
+`;
+
+const QueryContainer = styled.div`
+  pre {
+    height: ${({ theme }) => theme.gridUnit * 40}px;
+    border: none !important;
```

Comment:
Ping me when you have a chance, we can pair up and see if there's a better selector to remove the need for `!important` here.